app/[locale]/globals.css

```
@@ -0,0 +1,133 @@
+  --accent-foreground: #1A0E05;
+  --destructive: oklch(0.577 0.245 27.325);
+  --border: #D4C4AE;
+  --input: #D4C4AE;
+  --ring: #C4922A;
+  --radius: 0.75rem;
+  --chart-1: oklch(0.87 0 0);
+  --chart-2: oklch(0.556 0 0);
+  --chart-3: oklch(0.439 0 0);
+  --chart-4: oklch(0.371 0 0);
+  --chart-5: oklch(0.269 0 0);
+  --radius: 0.625rem;
```

Stylelint (17.14.0)

[error] Duplicate custom property "--radius" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)